sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -1167,10 +1173,9 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
       // When the assessment data is populated, the workflow is complete.
       // Only running workflow instances can be deleted.
       if (application.assessmentWorkflowId && !application.hasAssessmentData) {
-        const deleteAssessmentPromise =
-          this.workflowClientService.deleteApplicationAssessment(
-            application.assessmentWorkflowId,
-          );
+        const deleteAssessmentPromise = this.cancelAssessmentQueue.add({
```

Comment:
mm...curious about what happens when we wait for this?

Reply:
It will be awaited at line 1197.
Not sure if I followed the question but the `this.cancelAssessmentQueue.add` is an I/O operation similar to any other network/DB access.

Reply:
okay... so, we are waiting till the jobs are added to the queue. ..okay...ya.. that was my question